core/scripts/cli/sponsor/listMarkets.js

```diff
@@ -20,8 +22,37 @@ const listMarkets = async (web3, artifacts) => {
       emps.push(await ExpiringMultiParty.at(address));
     }
   }
-  // TODO: Show a better formatted and selectable table here.
-  console.log("LIST MARKETS:\n", emps);
+
+  // Format a useful display message for each market.
+  const backChoice = "Back";
+  const choices = [];
+  for (let i = 0; i < emps.length; i++) {
+    const emp = emps[i];
+
+    const tokenFactory = await emp.tokenCurrency();
```

Comment:
nit: rename to tokenAddress? It's not really the TokenFactory here, it's the ERC20

Reply:
Done.